miasm2/expression/expression.py

```diff
@@ -390,7 +393,8 @@ def __init__(self, arg, size):
         @arg: 'intable' number
         @size: int size"""
         super(ExprInt, self).__init__()
-        # Work is done in __new__
+        # Work for ._arg is done in __new__
```

Comment:
I think it's a bit confusion to set the `._arg` in the `__new__`, and the `._size` `__init__`.
Why such a code split?